examples/flutter-drm-backend/README.md

```diff
@@ -10,3 +10,11 @@ $ cd build
 $ cmake -DUSER_PROJECT_PATH=examples/flutter-drm-backend ..
 $ make
 ```
+
+## Running your Flutter app
+
+`FLUTTER_DRM_DEVICE` must be set properly when you use the DRM backend. The default value is `/dev/dri/card0`.
+
+```
+$ FLUTTER_DRM_DEVICE="/dev/dri/card1" ./flutter-drm-backend ./sample/build/linux/x64/release/bundle
```

Comment:
./sample/build/linux/x64/release/bundle -> FLUTTER_BUNDLE_PATH

Note that replace `FLUTTER_BUNDLE_PATH` with the flutter bundle path you want to use like `./sample/build/linux/x64/release/bundle`.